docs/pages/setup/reference/backends.mdx

```diff
@@ -265,6 +265,11 @@ teleport:
 
     # This setting configures Teleport to save the recorded sessions in an S3 bucket:
     audit_sessions_uri: s3://Example_TELEPORT_S3_BUCKET/records
+
+    # Teleport by default stores stores audit events with an AWS TTL of 1 year.
+    # This value can be configured as shown below and can be set to any integer number
+    # representing seconds. If set to 0, TTL is disabled.
+    audit_retention_period: 365 * 24 * 60 * 60
```

Comment:
Won't this also require changes to fileconf.go?

Should we be consistent with other configuration fields and allow duration values like `60m`?

Reply:
Changed that, it's now possible